changelogs/CHANGELOG-1.0.x.md

```diff
@@ -7,6 +7,9 @@
 - Fix bug where light nodes disconnect peers if the user queries the genesis block.
 - Add periodic heartbeat messages to the light protocol.
 
+## Bug Fixes
```

Comment:
This should go in to a new section `1.0.2` above the previous release (see [here](https://github.com/Conflux-Chain/conflux-rust/pull/1953/files)).